schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -667,6 +667,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_eventSources" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_referenceDataSets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments_accessPolicies" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-08-15-preview/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2018-08-15-preview/Microsoft.TimeSeriesInsights.json#/resourceDefinitions/environments" }, [](start = 18, length = 145)

we have a new root schema now: 2019-04-01/deploymentTemplate.json. can you please add them to the new root schema as well?

Reply:
Added